### PR TITLE
Provide an optional target url for git-set-commit-status

### PR DESCRIPTION
**Pull request by jparra5**

### CODE REVIEW

**Comment by jparra5** [comment]
@jauninb

File: git/task-set-commit-status.yaml [hunk 1]
```diff
@@ -282,6 +287,9 @@ spec:
         target_url = ("https://cloud.ibm.com/devops/pipelines/tekton/" + os.environ["PIPELINE_ID"] +
           "/runs/" + os.environ["PIPELINE_RUN_ID"] + "?env_id=ibm:yp:" + previous_step["TOOLCHAIN_REGION"])
 
+        if "$(params.target-url)" != "":
```
Comment (jauninb) [inline git/task-set-commit-status.yaml:290]: I would rather use an `if then else` to compute the target_url or only use it from the given parameter.
With this current code change, the default target_url is always computed even if there is a provided one

**Review by jauninb — CHANGES_REQUESTED** [review]
It would be better to use an if then else

File: git/task-set-commit-status.yaml (hunk 1 above)
Comment (jparra5) [reply git/task-set-commit-status.yaml]: @jauninb I have updated the PR.

**Review by jauninb — APPROVED** [review]
LGTM

**Pull request merged**